Bottom: 48f56b1ae7023d66928f21329d13d27c51de1707
Top:    9c2843f13567e3e5b9b87674f3a0d5613aa57fb7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 23:48:46 +0000

hid/gerber: Remove unused variables outline_trace_size and absolute

Left over accidentally from commit 35b137c3e193471e07f0873dfb149cebd059dca2


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index b7e99bb..8f04180 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -93,7 +93,6 @@ static int is_drill;
 static int current_mask;
 static int flash_drills;
 static int copy_outline_mode;
-static int outline_trace_size;
 static LayerTypePtr outline_layer;
 
 enum ApertureShape
@@ -444,7 +443,6 @@ gerber_do_export (HID_Attr_Val * options)
   static int saved_layer_stack[MAX_LAYER];
   int save_ons[MAX_LAYER + 2];
   FlagType save_thindraw;
-  bool absolute;
 
   save_thindraw = PCB->Flags;
   CLEAR_FLAG(THINDRAWFLAG, PCB);
